pkgs/packages/api/src/models/common/respiration.ts

```diff
@@ -6,4 +8,8 @@ export interface Respiration {
     avg_pct?: number;
   };
   avg_breaths_per_minute?: number;
+  /**
+   * Samples of respiration readings, breaths per minute over time.
+   */
+  samples?: Sample[];
```

Comment:
should we call this `samples_breaths_per_minute` - so units are explicit?

elsewhere we do this, ie `HeartRate.samples_bpm`: https://github.com/metriport/metriport/blob/661a76a16e9420ddb768b494c8625f239762838a/pkgs/packages/api/src/models/common/heart-rate.ts#L8

Reply:
I followed `BloodPressure`, which infers the unit from the property its in. Can do!

Reply:
Done!